### PR TITLE
Added the top navbar component

### PR DESCRIPTION
**Pull request by divitavora**

### CODE REVIEW

File: src/components/adminNavbar/AdminNavbar.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+class AdminNavbar extends Component {
+    
+    render() {
+        const dropdownName = "Hi, " + this.props.username;
+        return (
+            <div>
+                <Navbar bg="light" variant="light">
+                    <Navbar.Brand>
+                        <Link to="/admin">JJC Admin Panel</Link>
```
Comment (shah-tejas) [inline src/components/adminNavbar/AdminNavbar.js:15]: Can we add JJC logo here instead of plain text?

**Review by shah-tejas — CHANGES_REQUESTED** [review]
Can we add the JJC logo on the top left part instead of the plain text?

File: src/components/adminNavbar/AdminNavbar.js (hunk 1 above)
Comment (Nishit-Gopani) [reply src/components/adminNavbar/AdminNavbar.js:15]: I think that will be covered in the side navbar.

Comment (shah-tejas) [reply src/components/adminNavbar/AdminNavbar.js:15]: Okay, then this looks good!

**Review by shah-tejas — APPROVED** [review]

**Pull request merged**